quickwit-metastore/Cargo.toml

```diff
@@ -43,4 +44,4 @@ tempfile = '3'
 [features]
 testsuite = ["mockall", "tempfile"]
 ci-test = []
-postgres = [ "diesel", "diesel_migrations" ]
+postgres = [ "diesel", "diesel_migrations", "openssl"]
```

Comment:
Why don't use we `openssl/vendored` here?

Reply:
This helped more like selecting the ssl version to be same with the one on `quickwit/indexing`, I did not try `openssl/vendored` tbh. I think this might even be a hint to make build for `arm-musl` Will test it on https://github.com/quickwit-inc/quickwit/issues/715